tests/unit/test_get_translation.py

```diff
@@ -0,0 +1,123 @@
+import unittest
```

Comment:
Do we really need the 'unit' subdir?

Reply:
Introduced this when I started writing integration test. Have reverted.